packages/component-boilerplate/package.json

```diff
@@ -0,0 +1,56 @@
+{
+  "name": "@saasquatch/component-boilerplate",
+  "version": "1.0.0",
+  "private": false,
+  "description": "Boilerplate for writing web components for the SaaSquatch widget environments",
+  "source": "src/index.ts",
+  "main": "dist/index.js",
+  "exports": "./dist/index.modern.js",
+  "module": "dist/index.module.js",
+  "types": "dist/index.d.ts",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/saasquatch/program-tools.git"
```

Comment:
@sam-beveridge Should also have the directory such as `packages/component-boilerplate` in a `locatiionn?` field?